src/dymaptic.GeoBlazor.Core/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
Since this file was added to .gitignore, we actually also need to _delete_ it and commit, to make it disappear from the repo.